tests/custom_maven_install/unsafe_shared_cache_with_pinning_install.json

```diff
@@ -203,10 +198,15 @@
                 "url": "https://repo1.maven.org/maven2/org/codehaus/mojo/animal-sniffer-annotations/1.17/animal-sniffer-annotations-1.17.jar"
             },
             {
-                "coord": "com.google.guava:listenablefuture:jar:sources:9999.0-empty-to-avoid-conflict-with-guava",
```

Comment:
Example of fix: this was for some reason last instead of near the other `com.google.guava` dependencies.